docs/FAQ.md

```diff
@@ -25,7 +25,11 @@ Upgrade everything with:
 
     brew upgrade
 
-Or upgrade a specific formula with:
+Upgrade everything __including__ casks with `auto_update: true` and `version :latest`
```

Comment:
Don't think the various `__` usage added here aids readability and it's inconsistent with existing style